tests/tests_twobody/test_events.py

```diff
@@ -55,16 +55,13 @@ def f(t0, u_, k):
         du_ad = np.array([0, 0, 0, ax, ay, az])
         return du_kep + du_ad
 
-    rr, _ = cowell(
-        Earth.k,
-        orbit.r,
-        orbit.v,
+    method = CowellPropagator(events=events, f=f)
+    rr, _ = method.propagate_many(
+        orbit._state,
```

Comment:
Using `method.propagate_many` here to make it more explicit that we're testing a `CowellPropagator` method instead of using `Orbit.to_ephem`. But we could as well change all these for `orbit.to_ephem(method=CowellPropagator(...))`.